docker-entrypoint.sh

```diff
@@ -365,6 +418,19 @@ docker_setup_db() {
 		fi
 	fi
 
+	# To create replica user
+	local createReplicaUser=
+	local changeMasterTo=
+	local startReplica=
+	if validate_replication ; then
```

Comment:
^ `if [ -n "$MARIADB_MASTER_HOST" ]; then` and omit `validate_replication` entirely. The rest will be checked in `docker_verify_minimum_env` already